### ISSUE TITLE
[BUG] Erroneous Listening Posts entries appear to be reported as shown on ExTool

### ISSUE THREAD

**Issue by DaftMav** [canonn-science/EDMC-Canonn#153]
First of all I'm assuming any data shown on the ExTool website is coming from EDMC-Canonn? They have a link to this github at the bottom but afaik ExTool isn't exactly from/by Canonn so I'm not entirely sure if this is the correct place to report this. Their discord link is invalid so...

**Describe the bug**
There are several erroneous entries, if you look at the [recently found Listening Posts](http://elite.laulhere.com/ExTool/index.php?mode=spacepois&type=Listening%20Post&sort=-date), it has several entries where the 'near body' is in a completely different system from the 'system' listed it was found in. That is of course impossible...

First occurrence is on 07-05 and the most recent one on 09-04 so it's been a few months that this has happened, these are the bad entries as listed on ExTool:

System | Near body | Note
---|---|----
Crucis Sector DL-Y c29 | Heart Sector EB-X c1-12 7 | Known memorial LP in Heart sector, note these 2 systems are almost 7500 ly away from each other!
Tir | Colonia 4 & Jaques Station | Both are known LPs in system Colonia, not Tir
Pleiades Sector IH-V c2-10 | Maia A 1 | Known LP to Dominic’s Corner
D Scorpii | Col 285 Sector GN-H b11-0 | Known LP to Dav's Hope

I visited two nearby systems to check, the Crucis sector and the D Scorpii ones. There are no LPs in those systems and it appears all of the "near body" systems are the ones with an already known LP.

It may be possible the error is entirely on the side of the ExTool website instead fo the plugin reporting bad data. I suggest first checking if the same data is showing up in the Canonn database (which afaik for Listening Posts is not publicly available) or not.

The bug/issue may go beyond just the listening posts by the way, I just have used that section a lot while creating a listening posts spreadsheet and noticed the bad entries when I went to check for any new entries.

**Comment by NoFoolLikeOne** [comment]
It's likely this bug was introduced with horizons a while back where FSS entries were being written to the journal before the FSDJump event causing tools to get confused. This should be handled by the Canonn plugin. But there may be old incorrect data. 

I'm not sure what the status is of extool  these days. But we still send data there.

There is a dump of all FSS events from Canonn if you wanted to download and help clean up the data.

**Comment by DaftMav** [comment]
Just wanted to add I just noticed a bugfix on EDMC's changelog that might be related or the cause of the erroneous reports: https://www.reddit.com/r/EliteDangerous/comments/znc9tw/elite_dangerous_market_connector_570/j0gc2we/


> There is a dump of all FSS events from Canonn if you wanted to download and help clean up the data.

Sure, I could probably have a look at that and try to find bad entries.

I did already receive a dump of possible listening posts from SovereignWinter some months ago which might be the same dump..? I checked a lot of the systems that weren't on my LP list and noticed a lot of systems without an Listening Post almost always have at least one "Faction owned installation" instead, wondering if some of those are counted as LPs somehow. I kept a log of what I've checked with notes so if it turns out it's the same list that wouldn't be a ton of work to report back on.
